salt/kubelet/kubelet.service

```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Kubernetes Kubelet Server
+Documentation=https://github.com/GoogleCloudPlatform/kubernetes
+After=docker.service
+Requires=docker.service
+
+[Service]
+WorkingDirectory=/var/lib/kubelet
+EnvironmentFile=-/etc/kubernetes/node_labels
+EnvironmentFile=-/etc/kubernetes/config
+EnvironmentFile=-/etc/kubernetes/kubelet
+ExecStart=/usr/bin/hyperkube kubelet \
+	    $KUBE_LOGTOSTDERR \
+	    $KUBE_LOG_LEVEL \
+	    $KUBELET_API_SERVER \
+	    $KUBELET_ADDRESS \
+	    $KUBELET_PORT \
+	    $KUBELET_HOSTNAME \
+	    $KUBE_ALLOW_PRIV \
+        $KUBELET_NODE_LABELS \
+	    $KUBELET_ARGS
+Restart=on-failure
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
kubelet will read the value of `--node-labels` exactly once, at which point, future changes to the value of kubelets `--node-labels` will be ignored. So, if this value changes, we don't want to restart kubelet .. Because it gains us nothing. However, I agree we shouldn't be changing this file.. Another comment suggests an alternative.